src/Compatibility/Microsoft.DotNet.Compatibility/Microsoft.DotNet.Compatibility.csproj

```diff
@@ -6,7 +6,7 @@
     <IsPackable>true</IsPackable>
     <IsShippingPackage>true</IsShippingPackage>
     <StrongNameKeyId>Open</StrongNameKeyId>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>1.1.0</VersionPrefix>
```

Comment:
Separately we were discussing just calculating this based on the repo branding, is that possible?  Maybe we can address the stabilization issue in this PR too.

Reply:
We could do that, but I thought that we said we were going to do that on main. The only problem about doing that here, is that main today generates builds that are 2.0.0 so making that change here will start generating versions that are greater (in NuGet's comparisons) to main branch. If that's ok with you, I'd rather make that version change first in main branch, and then cherry pick back if we think it is necessary.